src/views/collection-view/index.tsx

```diff
@@ -69,7 +69,7 @@ function CollectionView({
 			<CollectionTutorialList
 				isOrdered={ordered}
 				tutorials={tutorials.map((t: ClientTutorialLite) =>
-					formatTutorialCard(t, slug)
+					formatTutorialCard(t, collection)
```

Comment:
`formatTutorialCard` now needs `{ id, slug }`, not just `slug`. Passing in the `collection` felt like the clearest update here.